src/styles.scss

```diff
@@ -33,8 +33,11 @@ $focus-overlay-transition: all 0.2s cubic-bezier(0, 1, 0, 1) !default;
 
   &-target {
     &,
-    &:focus,
-    &:focus:not(:focus-visible),
```

Comment:
are you sure you do not want to state this separately for browsers that support :focus-visible?

Reply:
This is actually not necessary.

`&:focus:not(:focus-visible)` will be triggered for mouse users ONLY if
  :focus-visible is supported on this platform.